### ISSUE TITLE
Vanilla OS CLI tool support in Command Line Interface Pages project

### ISSUE THREAD

**Issue by EmilySeville7cfg** [Vanilla-OS/apx#103]
Hi! Recently I started [Command Line Interface Pages](https://github.com/emilyseville7cfg-better-tldr) project. It aims to do the same thing as TlDr, but in a better and cleaner way. I've refreshed [apx](https://github.com/emilyseville7cfg-better-tldr/cli-pages/pull/16) and [abroot](https://github.com/emilyseville7cfg-better-tldr/cli-pages/pull/17) pages, but feel free to suggest what commands need support. 😄 

Page render is located [here](https://github.com/emilyseville7cfg-better-tldr/prototypes/tree/main/clip-view).

**Comment by kbdharun** [comment]
Thanks @EmilySeville7cfg , glad to see you here. I will DM you the commands in a couple of minutes.

**Comment by kbdharun** [comment]
Good morning, you can refer to the commands here https://github.com/tldr-pages/tldr/pull/9725 in my PR in tldr and in the apx repo, handbook in https://github.com/Vanilla-OS/apx and https://handbook.vanillaos.org/2023/01/11/install-and-manage-applications.html.

Tldr, Apx is distro agnostic and supports zypper, dnf, xbps, apt, aur flags for the containers and nix is a separate nix subcommand too (which isn't a container). 

In Vanilla, you don't need to use --apt for Ubuntu container as it is by default the host container. I.e apx install git, does it inside Ubuntu container.

The commands for installing is `apx install <package>` and similarly for other commands refer to Apx readme. There is a unique command export with a flag called `apx export --bin` which exports the binary to be used outside the container. 

For nix it is `apx nix init`, `apx nix install <package>` and `apx nix remove <package>`.

For more detailed flags, check out the cmd directory in repo.

**Comment by EmilySeville7cfg** [comment]
I've changed urls to point to the [official site docs](https://documentation.vanillaos.org/) in my PRs.

**Comment by kbdharun** [comment]
Thanks for adding apx page to CLIP. Once the other one is merged, I will close this issue.

**Comment by matbme** [comment]
It seems that the PR has been stale for a while and we added a few new features that aren't there (zypper and xbps, nix, --all, etc). Do we close the existing PR and make a new one or use the same PR to add these items?

**Comment by kbdharun** [comment]
> It seems that the PR has been stale for a while and we added a few new features that aren't there (zypper and xbps, nix, --all, etc). Do we close the existing PR and make a new one or use the same PR to add these items?

You mean in tldr? I will update my existing one and ask a fellow maintainer to merge it.

**Comment by kbdharun** [comment]
Also, This issue is completed as the pages were added.
